oda_api/api.py

```diff
@@ -1389,7 +1389,9 @@ def from_response_json(cls, res_json, instrument, product):
             if 'contours' in skmap.keys():
                 data.append(GWContoursDataProduct(skmap['contours']))
 
-        d = cls(data, instrument=instrument, product=product, request_job_id=res_json['job_monitor']['job_id'])
+        request_job_id = res_json.get('job_monitor', {}).get('job_id', None)
```

Comment:
@ferrigno reported an issue where this happened.

Ok for the warning.

Reply:
Ok, but it is not clear that returning None is the right solution in that case. It might be that the situation was not nominal, and we should raise an exception.
Could we see an example case?

Reply:
An exception is now raised